tests/test_io.py

```diff
@@ -0,0 +1,145 @@
+import unittest
+from pathlib import Path
+from random import shuffle
+
+import numpy as np
+from pydicom import dcmread
+import pytest
+
+from highdicom.io import ImageFileReader
+
+
+class TestImageFileReader(unittest.TestCase):
+
+    def setUp(self):
+        super().setUp()
+        file_path = Path(__file__)
+        self._test_dir = file_path.parent.parent.joinpath(
+            'data',
+            'test_files'
+        )
+
+    def test_construct_image_file_reader(self):
+        filename = '/tmp/foo.dcm'
+        reader = ImageFileReader(filename)
+        assert reader.filename == filename
+
+    def test_read_single_frame_ct_image_native(self):
+        filename = str(self._test_dir.joinpath('ct_image.dcm'))
+        dataset = dcmread(filename)
+        pixel_array = dataset.pixel_array
+        with ImageFileReader(filename) as reader:
+            assert reader.number_of_frames == 1
+            frame = reader.read_frame(0)
+            assert isinstance(frame, np.ndarray)
+            assert frame.ndim == 2
+            assert frame.dtype == np.int16
+            assert frame.shape == (
+                reader.metadata.Rows,
+                reader.metadata.Columns,
+            )
+            np.testing.assert_array_equal(frame, pixel_array)
```

Comment:
There's a multi-frame ct image in the pydicom test data that would make a nice test also. It has just two frames.

```python
from pydicom.data import get_testdata_file

filename = get_testdata_file('eCT_Supplemental.dcm')
```

Reply:
Nice! Addressed with 4ba4eb5e0c80aa2351e989555729845f48c6361f